pytorch_lightning/callbacks/quantization.py

```diff
@@ -16,12 +16,20 @@
 ^^^^^^^^^^^^
 
 """
+import copy
 import functools
-from typing import Any, Callable, Optional, Sequence, Union
+from typing import Any, Callable, Dict, Optional, Sequence, Union
 
 import torch
+from torch import Tensor
 from torch.quantization import QConfig
 
+try:
+    from torch.quantization import FakeQuantizeBase
+except ImportError:
+    # For torch 1.6 and 1.7.
+    from torch.quantization import FakeQuantize as FakeQuantizeBase
```

Comment:
Nit:

You can import `from pytorch_lightning.utilities.imports import _TORCH_GREATER_EQUAL_1_8`

```py
if _TORCH_GREATER_EQUAL_1_8:
     from torch.quantization import FakeQuantizeBase
else:
      from torch.quantization import FakeQuantize as FakeQuantizeBase
```